src/commands/load.ts

```diff
@@ -26,34 +27,48 @@ const getRom = async (file: Discord.MessageAttachment) => {
     return rom
 }
 
+
 const load: Command = {
     name: 'load',
     aliases: ['load_rom', 'loadrom', 'lr'],
     args: [],
     description: 'Load a rom attached in the message.',
+    optionalArgs: ['test'],
     execute: async (message: Discord.Message, args: string[]) => {
-        if (message.attachments.size < 1) {
-            return message.reply(`Please attach a rom of the game.`)
-        }
-
-        const rom = message.attachments.first()!
+        const loadTestRom: Boolean = !!process.env.DEBUG_MODE && args.includes('test');
 
-        let romBinary: ArrayBuffer
-        try {
-            romBinary = await getRom(rom)
-        } catch (e: unknown) {
-            const error = e as Error
-            return message.reply(error.message)
+        if (!loadTestRom && message.attachments.size < 1) {
+            return message.reply(`Please attach a rom of the game.`)
         }
 
         const guild = message.guild!
-
         const emulator = new ServerEmulator(guild)
+        let romName : string = "";
+        if (!loadTestRom){
+            const rom = message.attachments.first()!
+            let romBinary: ArrayBuffer
+            try {
+                romBinary = await getRom(rom)
+                emulator.loadROM(romBinary);
+                romName += rom.name;
```

Comment:
Why are you appending to the romName? It implies there is already some important info there, did you mean to = ?